lib/streamlit/elements/utils.py

```diff
@@ -31,3 +40,32 @@ def last_index_for_melted_dataframes(data):
             return data.index[-1]
 
     return None
+
+
+def check_callback_rules(
```

Comment:
`check` is a perfectly fine prefix convention for this, but we could also consider `assert` as a convention, since that is an existing term for "check a condition and catch fire if it fails". I'd be fine with either.

Reply:
I actually purposefully avoided usage of the prefix `assert_` for fear of it implying that it enforces rules using a real assertion over raising an exception/a warning in the streamlit UI

Will leave this as-is if you don't have a strong opinion on this